dotCMS/src/main/java/com/dotmarketing/db/commands/UpsertCommandFactory.java

```diff
@@ -0,0 +1,30 @@
+package com.dotmarketing.db.commands;
+
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.db.DbType;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Factory class for the UpsertCommand
+ * @author andrecurione
+ */
+public class UpsertCommandFactory {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)